frontend/src/App.js

```diff
@@ -36,6 +37,9 @@ function App() {
         {
           hasRole(currentUser, "ROLE_ADMIN") && <Route path="/admin/editcommons/:id" element={<AdminEditCommonsPage />} />
         }
+        {
+          hasRole(currentUser, "ROLE_ADMIN") && <Route path="/admin/showleaderboard/" element={<AdminShowLeaderboardPage />} />
```

Comment:
On further reflection, I think the leaderboard is going to be specific to a commons.  Accordingly, the route should probaby be parameterized by commons id, similar to the /play/:commonsId route.

That is, instead of `/admin/showleaderboard/` it should be `/admin/showleaderboard/:commonsId`

And probably, before the leaderboard is in place, it might be helpful to set up the placeholder page so that it gets information for the specific commons (similar to the play page, or the edit page for a commons) and shows, for example, the name of the commons.

Later, we'll add the leaderboard for that commons, but we should at least make sure that we've identified a particular commons.

And the navigation to the leaderboard page perhaps should be a button on the admin commons table, similar to the buttons for Edit and Delete.